include_defines/ideal_MHD_defines.h

```diff
@@ -58,11 +58,15 @@ TODO: JANHUNEN_CENTRAL, BRACKBILL_BARNES_CENTRAL, cf.
 #define USE_SOURCE_GODUNOV_CENTRAL
 
 
-// Boundary Fluxes: USE_BOUNDARY_FLUX_HLL
-#define USE_BOUNDARY_FLUX_HLL
+// Boundary Fluxes: USE_BOUNDARY_FLUX_HLL USE_BOUNDARY_FLUX_LFF
+//#define USE_BOUNDARY_FLUX_HLL
+#define USE_BOUNDARY_FLUX_LFF
```

Comment:
It is more common to use the abbreviation `LF` for the Lax-Friedrichs flux, i.e. `USE_BOUNDARY_FLUX_LF` instead of `USE_BOUNDARY_FLUX_LFF`.